src/tabcenter.css

```diff
@@ -489,9 +501,21 @@ img, .tab *:not(.clickable) {
 /* DARK THEME CUSTOMIZATIONS */
 body.dark-theme {
     --tab-background-normal: 223, 15.2%, 18%;
-    --tab-background-active: 221, 41.4%, 53.1%;
-    --tab-background-hover: 222, 28.3%, 35.55%;
-    color: rgb(245, 247, 250);
+    --tab-background-active: 221, 41.4%, 33.1%;
+    --tab-background-hover: 222, 28.3%, 25.55%;
+    color: #c0c0c0; /*Lightened text to make it easier to read*/
```

Comment:
Drop the comments everywhere in the CSS unless you need to explain something the code doesn't.